azure-iot-device/tests/common/pipeline/test_pipeline_stages_base.py

```diff
@@ -3610,57 +3701,37 @@ def test_completes_all_waiting_connect_ops_on_transient_connect_exception(
                 op=op, error=transient_connect_exception
             )
 
-    @pytest.mark.parametrize("state", [pipeline_stages_base.ReconnectState.NEVER_CONNECTED])
     @pytest.mark.it(
-        "Does not create a reconnect timer if the connection fails with a transient error"
+        "Does not create a reconnect timer if the connection fails with a transient error on a first-time connection"
```

Comment:
>transient error on a first-time connection [](start = 74, length = 42)

same comment as the first time connection case

Reply:
same as above.

---
In reply to: [437760653](https://github.com/Azure/azure-iot-sdk-python/pull/576#discussion_r437760653) [](ancestors = 437760653)